zeus/optimizer/batch_size/client.py

```diff
@@ -0,0 +1,115 @@
+from __future__ import annotations
+
+from abc import ABC, abstractmethod
+from uuid import UUID
+
+import pynvml
+import httpx
+
+from zeus.callback import Callback
+from zeus.job import Job
+from zeus.monitor import ZeusMonitor
+from zeus.monitor.energy import Measurement
+from zeus.optimizer.batch_size.server.models import (
```

Comment:
`models` should be under `zeus.optimizer.batch_size.models` or `zeus.optimizer.batch_size.common`. Also see comment below about shared constants like `GET_BATCH_SIZE_PATH`.